modules/caddyhttp/reverseproxy/selectionpolicies_test.go

```diff
@@ -658,6 +658,9 @@ func TestCookieHashPolicy(t *testing.T) {
 	if cookieServer1.Name != "lb" {
 		t.Error("cookieHashPolicy should set a cookie with name lb")
 	}
+	if cookieServer1.Secure != false {
```

Comment:
How about? Unless I'm missing something 

```suggestion
	if cookieServer1.Secure {
```